gradle/assemble.gradle

```diff
@@ -247,6 +247,7 @@ allprojects {
                         if (isRootProject) {
                             zipfileset(src: rootProject.configurations.runtime.files.find { it.name.startsWith('openbeans') }, excludes: 'META-INF/*')
                         }
+                        zap pattern: 'org.codehaus.groovy.vmplugin.v7.IndyInterface'
```

Comment:
normally the Groovy version, that does not use indy, means indy is not used for Groovy itself, but the user can still use it. You pullrequest will mean that this is no longer possible.  You have to at least also add the Java7 class, or else there will be class loading trouble. And the instantiation of that class would have to change. A bigger problem are the classes from org.codehaus.groovy.classgen.asm.indy, which do reference this class as well. Without further steps the groovy runtime and the compiler would get into trouble on a Java7 JVM or higher.

Reply:
What do you think about having two android jars one without classes that use invoke dynamic and one with them still? 

It looks like Android will be supporting it on ap's 26+. The issue is that Google added a check for classes using the invoke-polymorphic. Little more info here. https://issuetracker.google.com/issues/37567710

I am able to use proguard to remove the classes that cause the failure, but it increases build times greatly.